### DISCUSSION TITLE
Restrict access to images uploaded in Pull Request

### DISCUSSION THREAD

**Discussion by Kaz-su** [community/community#47561] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Although the URL of the image uploaded in a pull request is supposed to be unguessable, if the URL is known, anyone can access it.
We have a security policy that prohibits uploading images to the public space.
This makes it impossible to attach an image to a Pull Request, and this is hindering our review process.
The proposed solution is to migrate to GitLab. 😓 
However, I prefer to continue using GitHub for the developer experience.
If it were possible to restrict access to images uploaded in pull requests so that only Organization users could see them, that would solve the problem.

**Comment by junaid1460** [comment]
+1 we can to track public issues, considering people don't know what is sensitive information, this is a necessity
